rstream/client.py

```diff
@@ -65,6 +72,8 @@ def __init__(
         self._client_properties = {
             "product": "rmq-streams-client",
```

Comment:
Since you are here..! I would change it to: `RabbitMQ Stream`. It is implicit that is a client